PLUB/Sources/Views/Home/Clipboard/Model/BoardModel.swift

```diff
@@ -10,6 +10,12 @@ import Foundation
 /// 게시글 모델
 struct BoardModel {
   
+  /// 게시글 상세조회위한 피드아이디
+  let feedID: String = ""
```

Comment:
feedID는 `Int`로 설정해주세요!
```suggestion
  let feedID: Int
```

Reply:
변경했습니다 !!